config.toml

```diff
@@ -430,3 +430,36 @@ DefaultContentLanguage = "en"
   weight = 6
   name = "Twitter"
   url = "https://twitter.com/redox_os"
+
+[languages.pt]
+    weight = 0
```

Comment:
The weight of pt must be 5, based on where you have placed it, and you must increment the weight of languages lower in the list